css/style.css

```diff
@@ -323,3 +271,68 @@ th {
 	font-weight: bold;
 	border: none;
 }
+
+/* Checkbox css */
+.toggle-btn {
+  width: 80px;
+  height: 40px;
+  margin: 10px;
+  border-radius: 50px;
+  display: inline-block;
+  position: relative;
+  background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAyklEQVQ4T42TaxHCQAyENw5wAhLACVUAUkABOCkSwEkdhNmbpHNckzv689L98toIAKjqGcAFwElEFr5ln6ruAMwA7iLyFBM/TPDuQSrxwf6fCKBoX2UMIYGYkg8BLOnVg2RiAEexGaQQq4w9e9klcxGLLAUwgDAcihlYAR1IvZA1sz/+AAaQjXhTQQVoe2Yo3E7UQiT2ijeQdojRtClOfVKvMVyVpU594kZK9zzySWTlcNqZY9tjCsUds00+A57z1e35xzlzJjee8xf0HYp+cOZQUQAAAABJRU5ErkJggg==") no-repeat 50px center #e74c3c;
```

Comment:
What is this line doing exactly?

Reply:
Line 280 through 283? I think that's just the css for the checkbox switches default state. 🤔  I only added lines 325 through 336 in style.css to change some of the default styling of the bootstrap tooltip. Then I moved the checkbox css below the track list css to try and keep it consistent with the tooltip css.

Reply:
Ahh, sorry for the trouble.